dists/go-ipfs/go-ipfs-dist

```diff
@@ -79,8 +79,8 @@ dist_version() {
   cd "$cwd"
 
   # generate dist.json file
-  echo "---> generating dist.json"
-  ./mkdist.js "$version"
+  echo "---> generating config for $version"
+  ../../node_modules/.bin/gulp dist "--path=$(pwd)" "--version=$version"
```

Comment:
no, let's not _force_ users to use gulp. `mkdist.js` worked perfectly fine and only required people having node.